src/Config/Main.hs

```diff
@@ -195,6 +195,8 @@ defaultConfig =
       , _nmNewTag = "unread"
       , _nmDraftTag = "draft"
       , _nmSentTag = "sent"
+      , _nmHasNewMailSearch = "tag:inbox and tag:unread"
+      , _nmHasNewMailCheckDelay = Just (5 * 10000 * 60) -- ^ 5 seconds
```

Comment:
Thought about this and I think I'll better use a custom data type like: 
```Haskell
data Delay = Seconds Int
                       | Minutes Int
```
I can see myself forgetting a zero or two and then you have a thread polling notmuch like crazy.

Reply:
Good idea actually. I'll keep it unresolved until I've updated the patch with some tests.